src/shared/constants.h

```diff
@@ -141,3 +141,26 @@ const unsigned int ROBOT_CHIP_ANGLE_DEGREES = 45;
 // How many robots are allowed in each division
 const unsigned DIV_A_NUM_ROBOTS = 11;
 const unsigned DIV_B_NUM_ROBOTS = 6;
+
+
+// Arduino
+
+// UART baud rate used to communicate between system and arudino
+const long ARDUINO_BAUD_RATE = 115200;
+
+/*
+ * each estop message is one byte and is defined as follows
+ * bit 0 (least significant bit): estop state, a value of 1 is play, 0 is stop
+ * bit 1-7: set to 0
+ * any other message received is considered a EstopState::STATUS_ERROR
+ */
+const int ESTOP_MESSAGE_SIZE_BYTES = 1;
+
+const unsigned char ESTOP_PLAY_MSG = 1;
+const unsigned char ESTOP_STOP_MSG = 0;
+
+// product and vendor id for Arduino Uno Rev3 (retrieved from
+// http://www.linux-usb.org/usb.ids )
+#define ARDUINO_ID_LENGTH 5
+const char ARDUINO_VENDOR_ID[ARDUINO_ID_LENGTH]  = "2341";
```

Comment:
is there any USB property that we can program/flash onto the Arduino that uniquely identifies a thunderbots e-stop

this is nonblocking, just curious

Reply:
probably worth pushing off and taking an issue for it

Reply:
#2119